### ISSUE TITLE
[Bug] Xamarin.Forms Label underline decoration does not work on iOS 

### ISSUE THREAD

**Issue by KevinHu-au** [xamarin/Xamarin.Forms#9737]
### Description
Xamarin.Forms Label underline decoration does not work on iOS, but works fine on Android
### Steps to Reproduce

1.  Create any XF page with a Label and set Decoration="underline"
2. 
3. 

### Expected Behavior
Should have an underline for the Label element
### Actual Behavior
No underline for iOS
### Basic Information

- Version with issue: Xamarin.Forms 4.4.0.991265
- Last known good version: N/A
- IDE: Visual Studio for Mac community 8.4 (build 2657)
- Platform Target Frameworks: 
  - iOS: 10.0
  - Android: Android 10.0(Q)
  - UWP:   
- Android Support Library Version: 
- Nuget Packages:
- Affected Devices:

**Comment by jsuarezruiz** [comment]
I can reproduce the problem. 

```
<StackLayout>
    <Label
        Text="Label"/>
    <Label
        TextDecorations="Underline"
        Text="Underline Label"/>
</StackLayout>
```

[Issue9737.zip](https://github.com/xamarin/Xamarin.Forms/files/4260056/Issue9737.zip)

The repro sample works fine on Android:

<img width="398" alt="Captura de pantalla 2020-02-27 a las 8 24 33" src="https://user-images.githubusercontent.com/6755973/75421476-9f12b500-593a-11ea-918e-c1f872744445.png">

And the result on iOS:
<img width="368" alt="Captura de pantalla 2020-02-27 a las 8 18 30" src="https://user-images.githubusercontent.com/6755973/75421350-578c2900-593a-11ea-9068-4a6843736671.png">

**Comment by KevinHu-au** [comment]
This is my workaround:
                    <Grid>
                        <Label
                            x:Name="forgotPassordLink"
                            HorizontalOptions="Start"
                            Text="{helpers:Localize ForgotPassword}">
                        </Label>
                        <BoxView
                            HeightRequest="1"
                            WidthRequest="{Binding Source={x:Reference forgotPassordLink}, Path=Width}"
                            BackgroundColor="Gray"
                            HorizontalOptions="Start"
                            VerticalOptions="End"/>
                    </Grid>

**Comment by Mikilll94** [comment]
Duplicate of #8272

This is the old issue which should be fixed long time ago.

**Comment by KevinHu-au** [comment]
Checked on the latest Xamarin.Forms package. It is working fine now on iOS and Android

**Comment by hartez** [comment]
Confirmed, this is fixed as of 4.6. Thanks for the heads-up, @KevinHu-au!
